README.md

```diff
@@ -62,6 +62,20 @@ Suppose pull request number is 123 and it passes Travis CI. The deployment link
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
+## Code Documentation
+
+This project uses [compodoc](https://compodoc.github.io/website/guides/jsdoc-tags.html) for its documentation. The goal of this tool is to generate a documentation for all the common APIs of the application like modules, components, injectables, routes, directives, pipes and classical classes. 
```

Comment:
Please change it to ->  We are using [compodoc](https://compodoc.github.io/website/guides/jsdoc-tags.html) for documentation. The goal of this tool is to generate a documentation for all the common APIs of the application like modules, components, injectables, routes, directives, pipes and classical classes.
Compodoc supports [these](https://compodoc.github.io/website/guides/jsdoc-tags.html) JSDoc tags.

Reply:
👍